pipelines/tekton/build-container-image-pipeline/check-model-and-containerfile-exists.yaml

```diff
@@ -20,11 +20,13 @@ spec:
     script: |
       #!/usr/bin/env bash
 
+      set -Eeuo pipefail
+
       # Check model files exist
       if [ -n "$(params.modelRelativePath)" ]; then
-        ls -l model_dir/$(params.modelRelativePath)/$(params.model-name)/
+        ls -l model_dir-$(params.model-name)/$(params.modelRelativePath)$(params.model-name)/
```

Comment:
Changes will likely be needed in the "copy-model-from-pvc" task for this also, since that assumes `model_dir`. Also then the corresponding code in `pipelines/model-upload-Makefile`.